src/main/java/org/mitre/dsmiley/httpproxy/ProxyServlet.java

```diff
@@ -650,6 +652,9 @@ protected void copyResponseEntity(HttpResponse proxyResponse, HttpServletRespons
             }
           }
         } finally {
+          if(proxyResponse instanceof CloseableHttpResponse) {
```

Comment:
insteanceof CloseableHttpResponse might better be instanceof Closeable to be less dependent on implementation details of that framework.

Reply:
Ok, I changed this to closable. It is already an implementation detail though as the blocking behavior when closing a chunked stream.